py3status/modules/rainbow.py

```diff
@@ -65,12 +66,16 @@
 import re
 import math
 from time import time
+HEX_RE = re.compile('#([0-9a-fA-F]{3}|[0-9a-fA-F]{6})')
 
 
 class Py3status:
-
+    """
+    """
+    # configuration parameters
     cycle_time = 1
     force = False
+    format = '{rainbow}'
```

Comment:
I think `{output}` would be better as that is what we are using for other containers `group` and `frame`.

We should document the placeholder in the docstring